evergreen/config.yml

```diff
@@ -867,6 +889,23 @@ buildvariants:
     distros:
     - rhel70-large
 
+- name: ubuntu-valgrind
+  display_name: "Ubuntu 2204 Valgrind"
+  run_on: ubuntu2204-large
+  expansions:
+    cmake_url: "https://s3.amazonaws.com/static.realm.io/evergreen-assets/cmake-3.20.3-linux-x86_64.tar.gz"
+    cmake_bindir: "./cmake_binaries/bin"
+    fetch_missing_dependencies: On
+    enable_valgrind: On
+    cmake_build_type: RelWithDebInfo
+    curl: "/opt/mongodbtoolchain/v3/bin/curl"
+    python3: "/opt/mongodbtoolchain/v3/bin/python3"
```

Comment:
is python3 not available from the system on ubuntu 2204?

Reply:
It looks like it is:
```
ubuntu@ip-10-128-110-235:~$ which python3
/usr/bin/python3
ubuntu@ip-10-128-110-235:~$ python3 --version
Python 3.10.6
```